### PR TITLE
 Need proper alignment in event's card #7001 

### LINKED ISSUE #7001: Need proper alignment in event's card
**Describe the bug**

There is no proper alignment in event's card
**To Reproduce**
Steps to reproduce the behaviour:
1. Go to eventyay.com
2. Click on the search bar
3. Type any event name

**Expected behaviour**

Need proper alignmrnt
**Screenshots**

Desktop View:
c5

Mobile View:
c6
**Desktop (please complete the following information):**
All

**Smartphone (please complete the following information):**
All

### PR DESCRIPTION
**Pull request by AryaArunB**
Fixes #7001

#### Short description of what this resolves:
Screenshot 2021-05-05 at 10 13 37 PM

#### Changes proposed in this pull request:

Screenshot 2021-05-05 at 10 56 21 PM

### CODE REVIEW

File: app/routes/account/billing/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-rout
 export default class extends Route.extend(AuthenticatedRouteMixin) {
   titleToken() {
     return this.l10n.t('Billing Info');
-  }
+  }// use an alternative
 
   beforeModel() {
```
Comment (maze-runnar) [inline app/routes/account/billing/index.js:9]: don't add comments

File: package.json [hunk 2]
```diff
@@ -41,10 +41,10 @@
     "@sentry/integrations": "^6.3.5",
     "@sentry/tracing": "^6.3.5",
     "@types/ember": "^3.16.5",
+    "@types/ember__test-helpers": "^2.0.0",
```
Comment (maze-runnar) [inline package.json:44]: no needed

File: app/routes/account/billing/index.js (hunk 1 above)
Comment (AryaArunB) [reply app/routes/account/billing/index.js:9]: sorry

File: package.json (hunk 2 above)
Comment (AryaArunB) [reply package.json:44]: okay

**Comment by MeghalBisht** [comment]
Handled in #7225

**Pull request closed without merging**